components/centraldashboard-angular/frontend/src/app/pages/iframe-wrapper/iframe-wrapper.component.ts

```diff
@@ -101,7 +112,18 @@ export class IframeWrapperComponent implements AfterViewInit, OnDestroy {
       if (currentUrl !== this.iframeLocation) {
         this.iframeLocation = currentUrl;
         const path = iframeWindow?.location.pathname;
-        const queryParams = this.getQueryParams(iframeWindow?.location.search);
+        let queryParams = getQueryParams(iframeWindow?.location.search);
+
+        /**
+         * Append current namespace to query parameters before using router.navigate
+         * since iframe's internal URL doesn't hold any information regarding the
+         * namespace and we want to prevent it from discarding this infromation from
+         * the Browser's URL.
+         */
```

Comment:
FYI https://google.github.io/styleguide/jsguide.html#formatting-block-comment-style. There is no need to use `/**`, you can change it to `/*`.